pkg/server/share_manager.go

```diff
@@ -105,6 +123,18 @@ func (m *ShareManager) Run() error {
 			}
 
 			m.logger.Info("Starting nfs server, volume is ready for export")
+
+			if err := m.getLease(); err != nil {
+				m.logger.WithError(err).Warn("Failed to get lease - no lease renewal will be done")
```

Comment:
There will be flood warning messages if users disable the fast failover feature.

Reply:
This is a similar supposition to the "goroutine will be launched repeatedly."  In fact, this will log one time, and then block in m.nfsServer.Run().